src/Summoner/License.hs

```diff
@@ -1,51 +1,81 @@
+{-# LANGUAGE LambdaCase #-}
```

Comment:
You don't need to write extension here. It's already in `default-extensions` in `.cabal` file.